src/assets/Standards/CollectibleStandards/ERC721/ERC721Standard.test.ts

```diff
@@ -16,9 +17,54 @@ describe('ERC721Standard', () => {
   beforeEach(() => {
     web3 = new Web3(MAINNET_PROVIDER);
     erc721Standard = new ERC721Standard(web3);
+    nock.disableNetConnect();
```

Comment:
Put this in a `beforeAll` instead of a `beforeEach` for consistency?

Reply:
accidentally force-pushed instead of making a new commit, but its addressed here now: https://github.com/MetaMask/controllers/pull/667/commits/3e48605dcb64c65ac23b6d21d10f06cbf2cb6d1a